index.src.html

```diff
@@ -513,20 +514,24 @@ <h3 id="fetch-integration">Fetch Integration</h3>
 
   ISSUE: Monkey patching! Talk with Anne.
 
-  If the <a>`Clear-Site-Data`</a> header is present in an HTTP
-  <a>response</a>, then data MUST be cleared before rendering the response to
-  the user. That is, before step #9 in the current <a>main fetch</a> algorithm,
-  execute the following step:
+  If the <a>`Clear-Site-Data`</a> header is present in an HTTP <a>response</a>
+  received from the network, then data MUST be cleared before rendering the
+  response to the user. That is, after step #14 in the current
+  <a>HTTP-network fetch</a> algorithm, execute the following step:
 
-  9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+  15.  If |response|'s `credentials flag` is set, and |response|'s
+       <a>header list</a> contains a header named <a>`Clear-Site-Data`</a>, then
+       execute [[#clear-response]] of [[CLEAR-SITE-DATA]] on |response|.
```

Comment:
Done. Sorry, didn't notice that :)

I updated bikeshed, but still see the same warning regarding the <var>. I guess the change hasn't been rolled out yet? Well, if it will soon, we can still proceed, since it's a non-fatal warning.